packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -46,14 +46,31 @@ export function resolveNameForTitleMap({ schema, properties, value }) {
 	if (!schema.titleMap) {
 		return;
 	}
-
 	// Here we add a field side by side with the value
 	// to keep the title associated to the value
 	const valueIsArray = Array.isArray(value);
 	const uniformValue = valueIsArray ? value : [value];
 
 	const names = uniformValue
-		.map(nextValue => schema.titleMap.find(titleMap => titleMap.value === nextValue))
+		.map(nextValue => {
+			const titleMap = schema.titleMap;
```

Comment:
please use spread operator

Reply:
Don't be shy, use suggestions 😄 

```suggestion
			const { titleMap } = schema;
```